etc/cloud/cloud.cfg.d/91-delphix-platform.cfg

```diff
@@ -60,3 +54,11 @@ datasource_list: [ None ]
 #
 apt:
   preserve_sources_list: true
+
+#
+# By default, ignore ssh keys passed from the cloud management console.
+# Also, discard any user data to prevent tempering with the default
+# Delphix configuration.
```

Comment:
It might also be useful to make a note here, that these configuration fields are specific to Delphix's cloud-init package/modifications. Otherwise, one might go looking in the cloud-init docs and not find these.